HIP/hip-513.md

```diff
@@ -87,32 +87,27 @@ Which will improve debugging and auditing.
  * The type of action described by the action proto.
  */
 enum ContractActionType {
-    /**
-     * default non-value.
-     */
-    NO_ACTION = 0;
```

Comment:
Please undo this change.  If NO_ACTION is not desired then it can be changed to UNKNOWN.  Zero values are elided from the wire and I don't want the meaning of the value incorrectly implied by an incomplete serialization.

This is also Google's recommended practice - https://developers.google.com/protocol-buffers/docs/style#enums

Reply:
[Here](https://github.com/hashgraph/hedera-protobufs/pull/211#discussion_r903857306) @steven-sheehy  suggested to change this enums, because they are populated by consensus nodes and just adds complexity in processing it since it's unclear how the mirror node should react to an UNKNOWN. On other hand I agree with you that enums with UNKNOWN is Google's recommended practice. Maybe we need to discuss which form to use

Reply:
DONE